setup.sh

```diff
@@ -323,13 +327,23 @@ function synchronize_files()
 
   # Copy kw main file
   mkdir -p "$binpath"
-  cmd_output_manager "cp $app_name $binpath" "$verbose"
-  ASSERT_IF_NOT_EQ_ZERO "The command 'cp $app_name $binpath' failed" "$?"
+  if [[ "$ENABLE_TRACING" == 1 ]]; then
+    sync_main_kw_file_with_tracing "$app_name" "$binpath" "$TRACING_CODE_EXCERPTS_DIR"
+    ASSERT_IF_NOT_EQ_ZERO 'Could not sync kw main file with tracing enabled' "$?"
+  else
+    cmd_output_manager "cp $app_name $binpath" "$verbose"
+    ASSERT_IF_NOT_EQ_ZERO "The command 'cp $app_name $binpath' failed" "$?"
+  fi
```

Comment:
Nitpick: I guess the first `if` will usually be false; in this sense, it might be better to reverse the logic.

Reply:
Done